tsconfig.config.json

```diff
@@ -3,7 +3,8 @@
   "extends": "@vue/tsconfig/tsconfig.json",
   "include": [
     "vite.config.*",
-    ".eslintrc"
+    ".eslintrc",
+    "./GithubStats.ts"
```

Comment:
ts files ares already included.
We will use `curl` to download the json.
The module only need to fetch the file and parse the data.